docs/production-workflows.md

```diff
@@ -6,11 +6,12 @@ There are three main workflows for production:
 
 ### Creating a Bundle
 
+***Important**: The module names such as `app/main` used in the examples below should only be `/` separated (On Windows, do NOT use `\` as your path separator for this argument). The module names are specified in URL space; in particular, they are not file-paths.*
```

Comment:
I think this should be reworded slightly. It should say something closer to what the title of this PR says. Also, I think **Note**: would be better than **Important**.